src/nagini_translation/translators/common.py

```diff
@@ -572,6 +575,40 @@ def get_function_call(self, receiver: PythonType,
                                   self.no_info(ctx), type, formal_args)
         return call
 
+    def get_function_call(self, receiver: PythonType,
+                          func_name: str, args: List[Expr],
+                          arg_types: List[PythonType], node: ast.AST,
+                          ctx: Context,
+                          position: Position = None) -> FuncApp:
+        """
+        Wrapper function to _get_function_call, handling receivers of
```

Comment:
IMO the docstring should say what the functionality is (so basically the current docstring of _get_function_call), and then the docstring of _get_function_call can say that it implements this functionality for non-union receivers or something like that. Otherwise one has to look up the docstring of _get_function_call, a private method, to find out what this one does.

Reply:
Agreed, changed as well. This enforces the "program to interface" principle also for documentation. Please revise if docstrings are ok now.